public/i18n/en-us.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Gulp task creates the files themselves but sure does not fill any translations for you. You must fill them manually